com.unity.template-hd/Packages/com.unity.template.hd/Documentation~/hd.md

```diff
@@ -14,7 +14,7 @@ This Project Template uses the following features:
 
 This Template utilizes the High Definition Render Pipeline.
 
-The High-Definition Render Pipeline prioritizes stunning, high-fidelity visuals and is intended for high-end hardware: GPU-compute-capable consoles, and PC hardware (DX11 and above, Xbox One, PS4, PS5, Metal (macOS),
+The High-Definition Render Pipeline prioritizes stunning, high-fidelity visuals and is intended for high-end hardware: GPU-compute-capable consoles, and PC hardware (DX11 and above, Xbox One, Xbox Series, PS4, PS5, Metal (macOS),
```

Comment:
Should this be **Xbox Series X, Xbox Series S,** instead of **Xbox Series,**?

Reply:
Xbox Series covers both machines, as they are a single platform.